src/styles/index.scss

```diff
@@ -1,3 +1,54 @@
+@import './utils/variables';
+
 body {
   margin: 0;
+  height: 100vh;
+  display: flex;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  margin: auto;
+  flex-wrap: wrap;
+  gap: $base-gap;
+  width: ($square-size * $days-in-week) + ($base-gap * ($days-in-week - 1));
+  padding: 10px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: $square-size;
+    width: $square-size;
+    background-color: $square-color;
+    border: 1px solid black;
+    transition: 500ms;
```

Comment:
Consider moving the `.calendar__day` styles to a separate file called `_calendar.scss` in the `blocks` folder. It's a good practice to separate the styles for each BEM block into a separate file. This makes the code more organized and easier to maintain.